docs/fastdds/xml_configuration/log.rst

```diff
@@ -27,13 +27,13 @@ These are described in the following table.
 +-------------------+---------------------------------------------------------------+-----------------------+----------+
 
 The following constitutes an example of an XML configuration file that sets the |Log-api| to use one
-|StdoutConsumer-api| and one |FileConsumer-api|:
+|StdoutConsumer-api|, one |StdoutErrConsumer-api| and one |FileConsumer-api|:
```

Comment:
```suggestion
|StdoutConsumer-api|, one |StdoutErrConsumer-api|, and one |FileConsumer-api|:
```

Reply:
Done in 2b7b8953a6a91a7f1e80fe972b8de90d78beffa0